CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ Remember to bring your dependencies up to date with `./scripts/venvinstall.sh` w
 
 - Minor: Add a global command cooldown module. This allows you to share a cooldown between selected commands. (#1714)
 - Minor: Add the QueUp module back. (#1570)
+- Minor: Added Open Graph metadata to improve look in embeds such as in Twitter and Discord. (#1721)
```

Comment:
According to the opengraph website you linked, Twitter uses it's own meta tags (not opengraph ones). Does it actually work for Twitter?